ArcGISRuntimeSDKQt_QMLSamples/Maps/DownloadPreplannedMap/DownloadPreplannedMap.qml

```diff
@@ -0,0 +1,284 @@
+
+                preplannedCombo.model = offlineMapTask.preplannedMapAreaList;
+                busy.visible = false;
+
+                for (let i = 0; i < offlineMapTask.preplannedMapAreaList.count; i++) {
+                    offlineMapTask.preplannedMapAreaList.get(i).loadStatusChanged.connect(function () {
+                        if (offlineMapTask.preplannedMapAreaList.get(i).loadStatus !== Enums.LoadStatusLoaded)
+                            return;
+
+                        if (i === 0)
+                            checkIfFileExists(i);
+
+                        var graphic = ArcGISRuntimeEnvironment.createObject("Graphic", { geometry: offlineMapTask.preplannedMapAreaList.get(i).areaOfInterest });
+
+                        graphicsOverlay.graphics.append(graphic);
+                    });
+                    offlineMapTask.preplannedMapAreaList.get(i).load();
+                }
+            }
+
+            onCreateDefaultDownloadPreplannedOfflineMapParametersStatusChanged: {
+                if (createDefaultDownloadPreplannedOfflineMapParametersStatus !== Enums.TaskStatusCompleted)
+                    return;
+
+                createDefaultDownloadPreplannedOfflineMapParametersResult.updateMode = Enums.PreplannedUpdateModeNoUpdates;
+                let result = createDefaultDownloadPreplannedOfflineMapParametersResult;
+
+                path = outputMapPackage +"/" + result.preplannedMapArea.portalItem.title + ".mmpk";
+                fileFolder.url = path;
+
+                if (fileFolder.exists) {
+                    var mmpk = ArcGISRuntimeEnvironment.createObject("MobileMapPackage", { path: path });
+                    mmpk.loadStatusChanged.connect(function () {
+                        if (loadStatus !== Enums.LoadStatusLoaded )
+                            return;
+
+                        if (mmpk.maps.length < 1)
+                            return;
+
+                        mapView.map = mmpk.maps[0];
+                        busy.visible = false;
+                    });
+                    mmpk.load();
+                    return;
+                }
+
+                var job = offlineMapTask.downloadPreplannedOfflineMapWithParameters(createDefaultDownloadPreplannedOfflineMapParametersResult, path);
+
+                job.jobStatusChanged.connect(function () {
+                    if (job.jobStatus === Enums.JobStatusFailed) {
+                        console.log(job.error.message + " - " + job.error.additionalMessage)
+                        busy = false;
+                        return;
+                    } else if (job.jobStatus !== Enums.JobStatusSucceeded) {
+                        return;
+                    }
+
+                    mapView.map = job.result.offlineMap;
+                    mapExists = true;
+                    busy.visible = false;
+                });
+
+                job.start();
+            }
+        }
+
+        Component.onCompleted: {
+            portalItem.load();
```

Comment:
I think the portalItem should get loaded by the map once it is set to the mapView